.circleci/config.yml

```diff
@@ -155,6 +163,51 @@ jobs:
               aws s3 cp "$pkg" "s3://pytorch/whl/nightly/" --acl public-read
             done
 
+  smoke_test_linux_conda:
+    <<: *smoke_test_common
+    steps:
+      - run:
+          command: |
+            set -x
+            source /usr/local/etc/profile.d/conda.sh && conda activate python${PYTHON_VERSION}
+            conda install -y -c pytorch-nightly torchaudio
```

Comment:
The PyTorch smoketests fix the version date, so they fail if the nightlies failed. Should we do that here? (I'm indifferent, just pointing out the discrepancy)

Reply:
Another option is to have the smoke test not run if the nightlies fail? Or do you have another suggestion?

Reply:
I'm not 100% sure how conda will resolve target versions, if it picks the latest version, then test here might still make sense. I'll spend sometime looking into this later